src/me/mrCookieSlime/Slimefun/SlimefunGuide.java

```diff
@@ -713,8 +713,8 @@ public void run(final Player p) {
 							index++;
 						}
 						else {
-							List<String> tooltip = Messages.local.getTranslation("tooltips.item-permission");
-						    menu.addItem(index, new CustomItem(Material.BARRIER, StringUtils.formatItemName(sfitem.getItem(), false), tooltip.toArray(new String[tooltip.size()])));
+							List<String> message = SlimefunPlugin.getItemCfg().getStringList(sfitem.getID() + ".permission-message");
```

Comment:
Why would you read from the config if you implemented a getter in SlimefunItem...??

Reply:
What about now?